src/routes/api/index.js

```diff
@@ -1,18 +1,7 @@
-const router = require("express").Router();
+import users from './users';
 
-router.use("/", require("./users"));
+const router = require('express').Router();
```

Comment:
@william000000 I think you can destructure router from express to as to follow ES6 principles like this `import { Router } from 'express';`